Client/src/Components/TabPanels/Account/ProfilePanel.jsx

```diff
@@ -179,7 +184,12 @@ const ProfilePanel = () => {
 
   // Initiates the account deletion process
   const handleDeleteAccount = () => {
-    //TODO - implement delete account function
+    dispatch(deleteUser(authToken)).then((action) => {
+      if (action.payload.success) {
+        dispatch(clearAuthState());
+        dispatch(clearMonitorState());
+      }
+    });
```

Comment:
Can we replace this with async/await?  

```
const action = await dispatch(deleteUser(authToken))
if (action.payload.success {
    ...
}
```

I always aim for the least amount of indentaiton possible